formats/take_part/publisher/details.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "image"
+  ],
+  "properties": {
+    "body": {
+      "type": "string"
+    },
+    "image": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri"
+        },
+        "alt_text": {
+          "type": "string"
+        }
+      }
+    },
+    "ordering": {
```

Comment:
This isn't required for publishing individual Take Part pages, the Get Involved page will still be rendered by Whitehall so will know the order in which to render them.